### PR TITLE
KEYCLOAK-18589 map-storage profile for KeycloakServer

### PR DESCRIPTION
**Pull request by hmlnarik**

### CODE REVIEW

File: testsuite/utils/pom.xml [hunk 1]
```diff
@@ -271,6 +271,34 @@
...
+        <profile>
+            <id>map-storage</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.codehaus.mojo</groupId>
+                        <artifactId>exec-maven-plugin</artifactId>
+                        <configuration>
+                            <systemProperties>
+                                <property><key>keycloak.profile.feature.map_storage</key><value>enabled</value></property>
```
Comment (mhajas) [inline testsuite/utils/pom.xml:283]:
```suggestion
                                <systemProperty><key>keycloak.profile.feature.map_storage</key><value>enabled</value></systemProperty>
```

Based on https://www.mojohaus.org/exec-maven-plugin/usage.html the `systemProperties` element should contain `systemProperty` elements. However, it works also with `property` so I am not sure this needs to be changed.

**Review by mhajas — DISMISSED** [review]
Thank you for the PR @hmlnarik. It looks good to me, I just added one comment however I am not sure it requires a fix so I am approving.

File: testsuite/utils/pom.xml (hunk 1 above)
Comment (hmlnarik) [reply testsuite/utils/pom.xml]: Thanks, good point.

**Review by mhajas — APPROVED** [review]
Thanks @hmlnarik, LGTM

**Pull request merged**